modules/client/examples/05-encoders-decoders/14-initialize-from.ts

```diff
@@ -0,0 +1,79 @@
+/* MARKDOWN
+---
+title: Apply Uninstallation
```

Comment:
`initializeFrom` will be called on an after an update, not an uninstallation. 
Inside `PluginSetupProcessor` update will call the `upgradeToAndCall` (here https://github.com/aragon/osx/blob/873cf252ec92e5f54662d0364e141e7b16348217/packages/contracts/src/framework/plugin/setup/PluginSetupProcessor.sol#L531, which then goes here https://github.com/aragon/osx/blob/873cf252ec92e5f54662d0364e141e7b16348217/packages/contracts/src/framework/plugin/setup/PluginSetupProcessor.sol#L671C20-L671C20 if `initData` is provided).

Accordingly, this example must be adapted to show an plugin update.